src/radical/pilot/agent/executing/flux.py

```diff
@@ -61,7 +63,8 @@ def initialize(self):
                                                   self.session.cfg,
```

Comment:
instead of `self.session.cfg` it should be `self._rm.info`

(`LaunchMethod.__init__(self, name, lm_cfg, rm_info, log, prof)`)

Reply:
related to this block
https://github.com/radical-cybertools/radical.pilot/blob/ea0f68fa9deec276363861188042a6ec18da0e2a/src/radical/pilot/agent/executing/flux.py#L62-L64

Reply:
@andre-merzky this one wasn't fixed yet

Reply:
Thanks, done now.